### ISSUE TITLE
Tooltip needs work for screen readers

### ISSUE THREAD

**Issue by 42tte** [SpareBank1/designsystem#504]
#### Which package is this issue related to?
ffe-form-react

#### Describe your issue (screenshots welcome!)
Example of bugs:
* aria-expanded on wrong element (content and not toggler), lacks aria-controls
* ? (question mark) lacks description. How would you feel clicking a button with a ? on it? It's like a random unexpected result. Add a explaining text, ie "Forklaring på {{inputLabel}}".
Example of enhancements:
* Move focus on click to opened content
* Add aria-describedby={{tooltipElemId}} to input element

#### Steps to reproduce
Use a screen reader on https://design.sparebank1.no/styleguidist/index.html#!/Tooltip
